src/Aspire.Hosting.Azure.SignalR/AzureSignalRExtensions.cs

```diff
@@ -47,6 +51,11 @@ public static IResourceBuilder<AzureSignalRResource> AddAzureSignalR(this IDistr
 
             service.Properties.Tags["aspire-resource-name"] = construct.Resource.Name;
 
+            // Supported values are Free_F1 Standard_S1 Premium_P1
+            service.AssignProperty(p => p.Sku.Name, new Parameter("sku", defaultValue: "Free_F1"));
+            // Supported values are 1 2 5 10 20 50 100
+            service.AssignProperty(p => p.Sku.Capacity, new Parameter("capacity", BicepType.Int, defaultValue: 1));
+
```

Comment:
Why are these parameters instead of being values?

Reply:
how is "being values" look like?

Reply:
```
            service.AssignProperty(p => p.Sku.Name, "'Free_F1'");
            service.AssignProperty(p => p.Sku.Capacity, "1");
```

Reply:
Ah I c, because I want users to change the value through code like:
```cs
builder.AddAzureSignalR("signalr")
            .WithParameter("sku", "Standard_S1")
            .WithParameter("capacity", 10);
```

Reply:
Instead, they can change it with the CDK directly.

Reply:
Cool, let me close the PR then.